test/app-luatest/gh_6200_lua_cjson_escapes_slash_test.lua

```diff
@@ -0,0 +1,10 @@
+local json = require('json')
+local t = require('luatest')
+local g = t.group()
+
+g.test_encode_link = function()
+    t.assert_equals(json.encode({url = "https://srv:7777"}), '{\"url\":\"https://srv:7777\"}')
+    t.assert_equals(json.encode("\ttable[]"), '\"\\ttable[]\"')
+    t.assert_equals(json.encode("\\"), "\"\\\\\"")
+    t.assert_equals(json.encode("/home/user/Code/tarantool"), "\"/home/user/Code/tarantool\"")
```

Comment:
1. I would use double square brackets (`[[blah blah]]`) for expected values to don't bother with escaping and make the text more readable.
2. `\t` (tab symbol) seems not related to the problem. Or I missed something?

Reply:
1. Got it.
2. The logic behind it was to check if for whatever reason some other escape sequences would stop working, otherwise the possible test cases a very limited.